hw_tensor_io.lua

```diff
@@ -0,0 +1,16 @@
+require 'io'
+require 'torch'
+require 'cutorch'
+
+local fname = 'data.kitti/x0.bin'
+local dim = torch.LongStorage {389, 1, 350, 1242}
+local sz = 1
+for i = 1, #dim do
+   sz = sz * dim[i]
+end
+print (sz)
+local binObj = torch.FloatStorage(fname, false, sz, false)
```

Comment:
It seems that we have to specify the size in order for the memory mapped storage to work.